site/envoy_versions_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package site
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/tetratelabs/getenvoy/internal/version"
+)
+
+const envoyVersionsPath = "./envoy_versions.json"
+
+// TestEnvoyVersionsJson ensures tarball URLs in the Envoy versions JSON appear correct.
+// These are not fetched to avoid causing excess load.
+func TestEnvoyVersionsJson(t *testing.T) {
```

Comment:
similar to the latest manifest checker, which will become useless except to support old versions of getenvoy as long as we choose to do that.